exercises/exercise5/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+)
+
+type Node struct {
+	Value int
+	Next  *Node
+}
+
+func Prepend(head **Node, x int) {
+	// Plan:
+	// * Create new head
+	// * Make current head new head's next
+	// * Addign new head to current head pointer
+
+	var newHead *Node = new(Node)
+	newHead.Value = x
+	newHead.Next = *head
+
+	*head = newHead
+}
+
+func Append(head **Node, x int) {
+
+	var current *Node = *head
+	for current.Next != nil {
+		current = current.Next
+	}
+
+	var appended *Node = new(Node)
+	appended.Value = x
+	appended.Next = nil
+	current.Next = appended
+}
+
+func main() {
+	var head *Node = new(Node)
```

Comment:
Please use idiomatic Go: `head := &Node{Value: 0}`

Reply:
OK, but it also looks like, that it needs to be changed in example too) https://github.com/prog-1/pointers/tree/main/exercises#:~:text=removed%20bool)-,Example%3A,-var%20head%20*

Reply:
OK, but it also looks like, that it needs to be changed in example too) https://github.com/prog-1/pointers/tree/main/exercises#:~:text=removed%20bool)-,Example%3A,-var%20head%20*